week-3/D-array-filter/exercise.js

```diff
@@ -8,7 +8,17 @@
 
 var pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-var pairsByIndex; // Complete this statement
+var pairsByIndex = pairsByIndexRaw.filter(function(indexes) {
+  // Complete this statement
+  if (
+    indexes !== null &&
+    indexes.length == 2 &&
+    indexes !== "Boolean" &&
+    indexes !== "string"
+  ) {
+    return indexes;
```

Comment:
This will work, but isn't exactly right. Remember that `.filter()` expects your callback function to return a boolean (`true` or `false`). Here we're returning the element in the array